src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiCustomUIOptions.cs

```diff
@@ -46,21 +48,34 @@ public virtual async Task<string> GetStylesheetAsync()
             }
         }
 
+        private async Task<string> ReadFromUri(Uri uri)
+        {
+            using (var client = new HttpClient())
+            {
+                var response = await client.GetAsync(uri);
+                response.EnsureSuccessStatusCode();
+                return await response.Content.ReadAsStringAsync();
```

Comment:
This `EnsureSuccessStatusCode()` is supposed to handle exceptions. We don't need exception handling here. Instead, you can use `GetStringAsync(uri)` so that it returns empty (or null) string if the GET method returns 400+ HTTP status code.